gmso/core/topology.py

```diff
@@ -1425,3 +1425,46 @@ def load(cls, filename, **kwargs):
 
         loader = LoadersRegistry.get_callable(filename.suffix)
         return loader(filename, **kwargs)
+
+    def convert_potential_styles(self, expressionMap={}):
+        """Convert from one parameter form to another.
+        Parameters
+        ----------
+        expressionMap : dict, default={}
+            map with keys of the potential type and the potential to change to
+
+        Examples
+        ________
+        # Convert from RB torsions to OPLS torsions
+        top.convert_potential_styles({"dihedrals": "OPLSTorsionPotential"})
+        """
+        from gmso.utils.conversions import convert_topology_expressions
+
+        return convert_topology_expressions(self, expressionMap)
```

Comment:
After discussing with @daico007, all of this will get refactored so that the 3 engine writers are using similar methods. This method is analagous to what HOOMD and GROMACS Top writers are currently using locally, so that code will end up being extracted to this topology method.